src/geode/io/image/raster_image_input.cpp

```diff
@@ -23,54 +23,75 @@
 
 #include <geode/io/image/private/raster_image_input.h>
 
-#define cimg_display 0
-#define cimg_use_jpeg
-#define cimg_use_png
-#include <cimg/CImg.h>
+#include <gdal_priv.h>
 
 #include <geode/basic/attribute_manager.h>
+#include <geode/basic/logger.h>
+
 #include <geode/image/core/raster_image.h>
 #include <geode/image/core/rgb_color.h>
 
 namespace
 {
+    absl::FixedArray< GByte > read_band( const geode::RasterImage2D& raster,
```

Comment:
what is a band? maybe change the name by using what it stands for and not the name in GDal

Reply:
looking at the code, it seems to be a color attribute